src/main/java/ch/wisv/chue/HueService.java

```diff
@@ -43,6 +46,10 @@
      * @param lightIdentifiers the lights to apply the state to
      */
     public void loadState(HueState state, String... lightIdentifiers) {
+        if (!hueFacade.bridgeAvailable()) {
+            throw new StateNotLoadedException("Hue bridge is not available");
```

Comment:
It would be nice to move this into e.g. an annotation, but that might be something for later.